config/database.yml.example

```diff
@@ -13,7 +13,7 @@ development:
 
 test:
  <<: *default
- database: sample_project_test
+ database: sample_project_test<%= ENV['TEST_ENV_NUMBER'] %>
```

Comment:
why is this change in the db name?

Reply:
@megatux this is part of adding this gem, and also we're going to merge this and other PRs into a `release` branch, so when we have the functionality working as expected we'll create a new PR to merge against main. That works for you?